Readme.md

```diff
@@ -22,7 +22,7 @@ The repository that is created from this template must have one of the following
 
```

Comment:
Do you mind adding references to Bamboo on Lines 5 and 16? (GH won't let me add comments to unchanged lines 😢 )

Reply:
Ah! Missed those - [1fc3b25](https://github.com/actions/importer-issue-ops/pull/28/commits/1fc3b251d6de1440d58877224f34b5519b3b31fb)